R2000:
  transport: "udp"
  scanner_ip: "192.168.205.30"
  device: "R2000"
  start_angle: -1800000
  max_num_points_scan: 0
  packet_type: "B"
  scan_topic: "/scan"
  frame_id: "scanner_link"
  watchdogtimeout: 60000
  watchdog: true
